src/ripple/app/misc/AmendmentTable.h

```diff
@@ -154,10 +211,26 @@ class AmendmentTable
     doVoting (Ledger::ref lastClosedLedger, SHAMap::ref initialPosition) = 0;
 };
 
-std::unique_ptr<AmendmentTable>
-make_AmendmentTable (std::chrono::seconds majorityTime, int majorityFraction,
-    beast::Journal journal);
+namespace AmendmentTableDetail
```

Comment:
`namespace detail` is good enough, no need to invent a new identifier for every detail namespace.